Makefile

```diff
@@ -91,8 +86,19 @@ vet:
 # Generate code
 .PHONY: generate
 generate: $(CONTROLLER_GEN)
+	$(MAKE) generate-manifests
+	$(MAKE) generate-deepcopy
+
+# Generate deepcopy files.
+.PHONY: generate-deepcopy
```

Comment:
this is missing the actual target name

needs a `generate-deepcopy:` on the next line